addons/block-cherry-picking/addon.json

```diff
@@ -30,6 +26,7 @@
     {
       "name": "Flip controls",
       "id": "invertDrag",
+      "description": "Make grabbing blocks individually the default behavior. Hold Ctrl to drag entire stacks.",
```

Comment:
You can keep this here or remove it before merging, but if you do keep it, wait until the manifest schema and translation system are updated to include this new property before merging.

Reply:
I readded it to make sure translating this new field works properly. I didn't check the schema though